virtio-devices/src/net.rs

```diff
@@ -587,6 +587,7 @@ impl VirtioDevice for Net {
         interrupt_cb: Arc<dyn VirtioInterrupt>,
         mut queues: Vec<Queue<GuestMemoryAtomic<GuestMemoryMmap>>>,
         mut queue_evts: Vec<EventFd>,
+        _resample_evt: Option<EventFd>,
```

Comment:
Not all devices need resampling. In my tests, resampling was only seen in the first interrupt of virtio-blk device. So I am confused on what type of interrupt level the devices are using (level-triggerd or edge-triggered). 

Please comment if you know about this.

Reply:
For your reference:
https://patchwork.kernel.org/project/kvm/patch/20120921175601.32604.63271.stgit@bling.home/